### PR TITLE
Twilio messaging SID update

### PR DESCRIPTION
**Pull request by escopecz**
Documentation update for https://github.com/mautic/mautic/pull/12539

### CODE REVIEW

**Comment by escopecz** [comment]
@dadarya0 could you please review if these are the changes needed?

**Comment by RCheesley** [comment]
Sorry team but these docs are really inadequate I feel based on the experience I just had trying to figure out how to get the SID.

There is a whole new UI with multiple steps and settings that need to be explained before you get anywhere near getting the SID.

screenshot-console twilio com-2023 08 04-18_22_51

screenshot-console twilio com-2023 08 04-18_23_14

screenshot-console twilio com-2023 08 04-18_23_27

screenshot-console twilio com-2023 08 04-18_23_50

File: docs/docs_translations/plugins.pot [hunk 1]
```diff
@@ -117,7 +117,7 @@ msgid "Open *Twilio* Plugin and activate it."
 msgstr ""
 
 #: ../plugins/twilio.rst:23
-msgid "Copy the *Account Sender ID* from Twilio account and paste it to *Account Sender ID* field in the Twilio Plugin configuration."
+msgid "Copy the *Messaging Sender ID* from Twilio account and paste it to *Messaging Sender ID* field in the Twilio Plugin configuration."
```
Comment (dadarya0) [inline docs/docs_translations/plugins.pot:120]: @escopecz Account sender ID is different thing we had added Messaging Service SID and it will be used in place of phone number in feature tab.
So Account Sender ID will be remain as it is and phone number will be replace by Messaging Service SID.

Comment (escopecz) [reply docs/docs_translations/plugins.pot:120]: @dadarya0 can you rephrase the message for me?

Comment (escopecz) [reply docs/docs_translations/plugins.pot:120]: Or even create a new PR with the changes needed? I don't get the change clearly.

Comment (dadarya0) [reply docs/docs_translations/plugins.pot:120]: In feature setting of Twilio configuration, **Sending Phone Number** is replace by **Messaging Service SID**.

image

**How to generate Messaging Service SID :-**

1. Login to twilio account
2. In search box, serach for Messaging services  and select Messaging services from search result

image


3. Click on Create Messaging Service button

4. Input all required field and configure it

5. Now user will get Messaging Service SID, now copy and use it in acs.

Comment (RCheesley) [reply docs/docs_translations/plugins.pot:120]: This whole process, including the explanation of 'input all required fields and configure it' step by step, is needed for this PR to be mergeable.

**Review by RCheesley — CHANGES_REQUESTED** [review]
Hi folks,

We can't merge this without adequate information guiding the user how to get the SID. I'd suggest going step by step with screenshots - @fakela do you have any capacity to help getting this over the line?

**Comment by RCheesley** [comment]
Closing this PR as I feel we need to actually re-write the documentation and this now has a bunch of conflicts to resolve.
